CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Remove: local mergeDeviceWithConfiguration redefined from iota-node-lib (#642)
 - Fix: ensure service and subservice from device in logs about error proccesing message
 - Fix: use but not store timestamp and entityNameExp from group with autoprovisioned devices (iotagent-node-lib#1504, partially)
```

Comment:
```suggestion
```

As far as I remember that change was a modification in local mergeDeviceWithConfiguration. Given that local mergeDeviceWithConfiguration disappears in this PR (in favour of the mergeDeviceWithConfiguration in the iotagent-node-lib) I think this line can be removed.